apiary.apib

```diff
@@ -520,21 +521,17 @@ List all items in the support Queue for a certain `CLIENT_KEY`.
 
 | Name | Description | Example |
 | ---------------| ------------| --------|
-| start time | | 1 |
-| session | | 1 |
-| url | | "" |
-
-    "start_time": 1,
-    "session": 1,
-    "url": ""
+| start time | | 2016-08-11T13:44:58.025358Z |
```

Comment:
according to our code, it should return "start_time", "url", 'start_url', "pin", "identifier" :/